content/en/apps/guides/data/analytics/building-dbt-models.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "DBT models for CHT Applications"
+linkTitle: "DBT Models"
+weight: 2
+description: >
+  Guide for building DBT models for CHT applications
+relatedContent: >
+  core/overview/db-schema
+  apps/reference/app-settings/hierarchy
+---
+
+## Overview
+
+[CHT Sync]({{< relref "core/overview/cht-sync" >}}) copies data from CouchDB to a relational database. It initially stores the document data from CouchDB in a `jsonb` column in a single table. This is not possible to query for analytics, so it uses [DBT](https://www.getdbt.com/) to convert the document data to a relational database format.
+
+[CHT Pipeline](https://github.com/medic/cht-pipeline) defines a DBT project, which contains model files for the data schema described [in the Database schema conventions]({{< ref "core/overview/db-schema" >}}).
+Forms may be specific to each CHT application; additional models will need to be developed to analyze data from responses to these custom forms.
+One additional model will be needed for each form, and for any aggregations, dashboards, or reusable views that use those form responses as input.
+If using the [configurable contact hierarchy]({{< ref "apps/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}), it may also be useful to add models for other contact types.
+
+## Setup
+
+To create application specific models, create a new dbt project as described [here](https://docs.getdbt.com/reference/commands/init)  and a Github repository (it may be public or private).
+The [CHT Pipeline](https://docs.getdbt.com/docs/build/packages) dbt project, should be included as a dependency in `packages.yml`
+```yml
+packages:
+  - git: "https://github.com/medic/cht-pipeline"
+    revision: "1.0.0"
+```
+To avoid breaking changes in downstream models, include a version tag in the dependency.
+
+In CHT Sync config, set the URL of this repository to the `CHT_PIPELINE_BRANCH_URL` [environment variable]({{< relref "apps/guides/data/analytics/environment-variables" >}}), either in ´.env´ if using ´docker compose´, or in ´values.yaml´ if using Kubernetes.
```

Comment:
```suggestion
In CHT Sync config, set the URL of dbt GitHub repository to the `CHT_PIPELINE_BRANCH_URL` [environment variable]({{< relref "apps/guides/data/analytics/environment-variables" >}}), either in ´.env´ if using ´docker compose´, or in ´values.yaml´ if using Kubernetes.
```

Reply:
I would add extra clarity here, by mentioning the need to create a PAT so that CHT Sync can access the models. That is mentioned in the env variables page, but could be added here too.